hazelcast/src/main/resources/hazelcast-default.xml

```diff
@@ -116,6 +116,12 @@
         <!--Queue capacity. 0 means Integer.MAX_VALUE.-->
         <queue-capacity>0</queue-capacity>
     </executor-service>
+    <scheduled-executor-service name="default">
```

Comment:
Similar comments for the `hazelcast-default.yaml`

Reply:
done.

Reply:
`<statistics-enabled>true</statistics-enabled>` is missing fot the `<executor-service name="default">`

Reply:
i hope it is fixed now.